containers/images/redis-sentinel/README.md

```diff
@@ -0,0 +1,66 @@
+# Redis sentinel packaged by SourceMation
+
+Redis is a performant in-memory key-value database. Its simplicity, accompanied
+by optimal data structures, contribute to the high operational speed.
+
+Built by SourceMation's automation team, this Redis release undergoes regular
+updates, following the official `stable` release. The foundation is the latest
+`sourcemation/debian-12-slim` image at build time, ensuring a compact, secure,
+and current setup.
+
+## Usage
+
+Run a temporary container with Redis, ready to accept connections from your
```

Comment:
It will start redis-sentinel, I'm not sure if it accept connections in the same way :)